### DISCUSSION TITLE
Bundling

### DISCUSSION THREAD

**Discussion by brnkgabriel** [StaticJsCMS/static-cms#861] [Q&A]
Hello guys,

Has anyone successfully gotten the bundling option to work. Seems to give one issue or the other. Tried with Svelte as well as Astro but it was one error or the other.

Will be grateful to have a template that has the bundling option as per the guide: https://www.staticcms.org/docs/add-to-your-site-bundling

Thanks in advance

**Comment by KaneFreeman** [comment]
The next template shows bundling in action: https://github.com/StaticJsCMS/static-cms-next-netlify-template
